graphql-kotlin-schema-generator/src/main/kotlin/com/expedia/graphql/generator/SchemaGenerator.kt

```diff
@@ -76,8 +75,8 @@ open class SchemaGenerator(val config: SchemaGeneratorConfig) {
     open fun listType(type: KType, inputType: Boolean) =
         listTypeBuilder.listType(type, inputType)
 
-    open fun objectType(kClass: KClass<*>, interfaceType: GraphQLInterfaceType? = null) =
-        objectTypeBuilder.objectType(kClass, interfaceType)
+    open fun objectType(kClass: KClass<*>) =
```

Comment:
Since we no longer directly call the object type, we never call the method passing in an interface so we can clean this up